oort/oort-aws/src/main/groovy/com/netflix/spinnaker/oort/aws/provider/agent/ClusterCachingAgent.groovy

```diff
@@ -398,7 +398,20 @@ class ClusterCachingAgent implements CachingAgent, OnDemandAgent, AccountAware,
     def usableOnDemandCacheDatas = []
     providerCache.getAll(ON_DEMAND.ns, asgs.collect { Keys.getServerGroupKey(it.autoScalingGroupName, account.name, region) }).each {
       if (it.attributes.cacheTime < start) {
-        evictableOnDemandCacheDatas << it
+        if (account.eddaEnabled) {
```

Comment:
we may want to look at reworking this to collect a list of ASG names and do the loadAutoScalingGroup as one call with all the ASGs we want to compare